lib/open_weather/endpoints/stations.rb

```diff
@@ -10,6 +10,19 @@ def register_station(options = {})
       def list_stations
         get('stations').map { |data| OpenWeather::Models::Station.new(data) }
       end
+
+      def get_station(id)
+        OpenWeather::Models::Station.new(get("stations/#{id}"))
+      end
+
+      def update_station(id, options = {})
+        OpenWeather::Models::Station.new(put("stations/#{id}", options))
```

Comment:
I'm not sure why, but it seems like the API requires `external_id` to be specified 🤷

Reply:
Also, I could potentially raise an error if `id` was not defined, do you have a suggestion on how to define / where to keep errors?

Reply:
If a parameter is missing, raise ArgumentError, generally I don't specialize errors unless they can happen on all APIs (e.g. incorrect HTTP proxy)